#Grupo de obstaculos detectados en un frame

Obstaculo[] grupo_obstaculos



